recipes/thrift/all/conanfile.py

```diff
@@ -161,7 +146,7 @@ def package(self):
             targets.update({"thriftz::thriftz": "thrift::thriftz"})
```

Comment:
Not related tho this pr, but should we add `lib/cmake` to `self.cpp_info.components["libthrift"].builddirs`?

Reply:
No, files created in lib/cmake for this recipe are internal details of cmake_find_package